lesson9.tex

```diff
@@ -0,0 +1,149 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{9}
+\title{Lesson \lessonNumber: [Ciphers]}
+\author{\linkHome}
+\date{}
+
+% Buildup
+\setLessonNumber{\lessonNumber}
+
+
+%Document
+\begin{document}
+
+  \maketitle
+
+  \section{Overview}
+    \subsection{Summary}
+      Here we give an overview of the Lesson and what it teaches, to
+      both younger and older audiences
+
+      This lesson teaches students how to crack and create their own
+      ciphers through the context of an ancient treasure.
+
+      For more advanced students, this lesson goes into basic mathematics 
+      of prime number and modulo based ciphers. 
+
+    \subsection{Objectives}
+    \begin{itemize}
+      \item Introduce students to basic letter ciphers
+      \item (Intermediate) Introduce the concept of morse code and modern cyber security.
+      \item (Advanced) Describe the concept of prime number factorization for use in security systems.
+      \item (Cooperative) Have separate groups work together to help solve an ancient cipher!
+    \end{itemize}
+    \subsection{Key Vocabulary Terms}
+    \begin{itemize}
+      \item List some of the key takeaways from the lesson.  These are
+        also good things to ask about at the conclusion of the lesson
+      \item Cipher, Encryption, Decryption
+      \item (Intermediate) Cyber-security, Modulo
+      \item (Advanced) Prime-number factorization, RSA
+    \end{itemize}
+    \subsection{Materials}
+      \begin{itemize}
+        \item 2x Chromebooks for the group to work with for the Morse Code portion.
+        \item Cipher Wheel
+        \item 2x Tubes of certain diameters (One works with Scytale), hole drilled at end and tied to wooden base.
+        \item Alien language chart
+        \item 2x Morse chart
+        \item 2x USB with morse data on it (labeled 'part-1' and 'part-2', both also have morse code chart png).
+        \item 2x computer with ability to play sound and take USB
+        \item 2x treasure box (first and last puzzles)
+        \item Black light pen/light
+      \end{itemize}
+  \section{Procedure}
+    \subsection{Setup (10 minutes)}
+      \begin{itemize}
+        \item Place cipher wheel and caesarian cipher code in first box, labelled '1'.
+        \item Place Scytale strips in location revealed by caesarian cipher
+        \item Place scytale bases in different locations, revealed by notes in Box 1
+        \item Scytales produce 2-part descriptions of where next clue is (depending on location).
+            Examples could be 'opposite' 'window'. Place USBs at those locations.
+        \item Lock must be set to work with decided on number code (from Morse).
+        \item Place "Gold Coins" chocolate in the second box, labelled '2'. 
```

Comment:
\` for open left, ' for close right. Also not sure why this is wrapped in " here